session/driver/file_test.go

```diff
@@ -0,0 +1,125 @@
+package driver
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/goravel/framework/support/carbon"
+	"github.com/goravel/framework/support/file"
+)
+
+type FileTestSuite struct {
+	suite.Suite
+}
+
+func TestFileTestSuite(t *testing.T) {
+	suite.Run(t, &FileTestSuite{})
+}
+
+func (f *FileTestSuite) BeforeTest() {
+	f.Nil(file.Remove(f.getPath()))
+}
+
+func (f *FileTestSuite) TestClose() {
+	driver := f.getDriver()
+	f.Nil(driver.Close())
+}
+
+func (f *FileTestSuite) TestDestroy() {
+	driver := f.getDriver()
+
+	f.Nil(driver.Destroy("foo"))
+
+	f.Nil(driver.Write("foo", "bar"))
+	value, err := driver.Read("foo")
+	f.Nil(err)
+	f.Equal("bar", value)
+
+	f.Nil(driver.Destroy("foo"))
+
+	value, err = driver.Read("foo")
+	f.NotNil(err)
+	f.Equal("", value)
+}
+
+func (f *FileTestSuite) TestGc() {
+	driver := f.getDriver()
+
+	f.Nil(driver.Gc(300))
+
+	f.Nil(driver.Write("foo", "bar"))
+	f.Nil(driver.Gc(300))
+	value, err := driver.Read("foo")
+	f.Nil(err)
+	f.Equal("bar", value)
+
+	carbon.SetTestNow(carbon.Now(carbon.UTC).AddSecond())
+	f.Nil(driver.Write("baz", "qux"))
+	carbon.UnsetTestNow()
```

Comment:
`SetTestNow` first, then `UnsetTestNow` immediately, I may guess what you want to do, but I think it's unnecessary, it's a bit difficult to understand. the line 57, 59 can be removed.